backend/schemas/output/excel/json/federal-awards-expended-template.json

```diff
@@ -302,7 +310,11 @@
                "title_cell": "S1",
                "type": "open_range",
                "validation": {
-                  "type": "NOVALIDATION"
+                  "allow_blank": "True",
+                  "custom_error": "The Audit Report Type must be empty if Major Program is \"N\"",
+                  "custom_title": "Invalid Audit Report Type",
+                  "formula1": "=IF(R{0}=\"Y\",AuditReportTypes!$A$2:$A$5,AuditReportTypes!$A$6:$A$6)",
```

Comment:
Should we add a named range to this workbook for the A2:A5 and A6:A6 ranges? We can add named ranges even if they're not used for extraction, and that makes our formulas cleaner. Thoughts?